src/clients/starknet-tx/index.ts

```diff
@@ -8,59 +8,157 @@ import {
 } from 'starknet';
 import vanillaAbi from './abi/auth-vanilla.json';
 import ethSigAbi from './abi/auth-eth-sig.json';
-import constants from './constants.json';
+import constants from './constants';
 import * as utils from '../../utils';
+import { getAuthenticatorType, getStrategyType } from './contracts';
 import {
   EthSigProposeMessage,
   EthSigVoteMessage,
   VanillaVoteMessage,
   VanillaProposeMessage,
-  Envelope
+  Message,
+  Envelope,
+  Metadata
 } from '../../types';
 
 const { getSelectorFromName } = hash;
 
+const TEMP_CONSTANTS = {
+  strategyParams: ['0xB4FBF271143F4FBf7B91A5ded31805e42b2208d6', '0x3'],
+  block: 7529615
```

Comment:
If you want to start new proposal and vote on it, this will need to match block returned by `get_latest_l1_block` [here](https://goerli.voyager.online/contract/0x6ca3d25e901ce1fff2a7dd4079a24ff63ca6bbf8ba956efc71c1467975ab78f#readContract) **minus 1**.

Right now it matches, but if new block is pushed and processed it will need to be bumped.